src/components/checkboxes/index.md.njk

```diff
@@ -65,6 +65,24 @@ You can add hints to checkbox items to provide additional information about the
 
 {{ example({group: "components", item: "checkboxes", example: "hint", html: true, nunjucks: true, open: false, size: "s"}) }}
 
+### Add an option for ‘none’
+
+In some cases, it can be helpful to require users to select a checkbox to say that none of the options apply to them, instead of leaving them all unchecked. This also makes sure users do not skip the question by accident.
```

Comment:
Could make the wording active, to emphasise that it's users this helps. Maybe something like:

```suggestion
Sometimes, it helps users if you require them to check a box to say none of the options apply to them. Otherwise, they would have to leave all of the boxes unchecked. This requirement also makes sure users do not skip the question by accident.
```

Reply:
This slightly reads like this might only be helpful 'sometimes' - I wonder if it could be reframed that where 'none' is a valid answer, that should be offered as an explicit choice?

Reply:
Thanks, @edwardhorsford - specifying when it helps is a good idea. Could something like the below work?

```suggestion
Where 'none' is a valid answer, offer users the option to check a box to say none of the other options apply to them. Without this option, users would have to leave all of the boxes unchecked. This requirement also makes sure users do not skip the question by accident.```

Reply:
@frankieroberto Ah gotcha - thanks. Could something like the below work?
```suggestion
Give users the option to check a box to say none of the other options apply to them. Without this option, users would have to leave all of the boxes unchecked. This requirement also makes sure users do not skip the question by accident.
```

Reply:
Sorry, I think this also isn't right - we only want a 'none' option where it's a valid answer. Sometimes the checkboxes do cover all cases.

Example:
```
# Which of your training locations will the trainee visit?
Select all that apply
[ ] Location A
[ ] Location B
[ ] Location C
```

In this case, none wouldn't be right if they're always going to be visiting at least one of them.

Reply:
Thanks, @edwardhorsford - does the below work?

```suggestion
When 'none' would be a valid answer, give users the option to check a box to say none of the other options apply to them. Without this option, users would have to leave all of the boxes unchecked. This requirement also makes sure users do not skip the question by accident.
```

Reply:
@EoinShaughnessy that's much better 👍